app/src/main/java/de/westnordost/streetcomplete/controls/MainMenuDialog.kt

```diff
@@ -16,7 +16,7 @@ import kotlinx.android.synthetic.main.dialog_main_menu.view.*
 class MainMenuDialog(
     context: Context,
     onClickDownload: () -> Unit,
-    teamModeQuestFilter: TeamModeQuestFilter
+    val teamModeQuestFilter: TeamModeQuestFilter
```

Comment:
Either I pass the quest filter in here, so that I can access its `enableTeamMode`/`disableTeamMode` functions and `isEnabled`/`indexInTeam` properties, or I pass them as separate parameters. Do you prefer callbacks and separate parameters here, too?

Reply:
I have no preference here. Well... maybe slightly twards callbacks, because then the control-logic is all in one place (in MainMenuButtonFragment)